domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -1621,10 +1617,12 @@ public TotaledResults(ResultSet set, Integer total) {
     boolean returnIdField, boolean setId, List<FacetField> facets, String distinctOn,
     Handler<T> streamHandler, Handler<AsyncResult<Void>> replyHandler
   ) {
+    // streamGet appears to offer facets, but it does not implement it
```

Comment:
Can you move this comment into a javadoc for the facets parameter?
Can you reply with a failed future with an exception (for example UnsupportedOperationException) if facets is neither null nor empty?